js_modules/dagit/packages/core/src/workspace/useRepositoryForRun.ts

```diff
@@ -141,11 +141,11 @@ export const useRepositoryForRunWithParentSnapshot = (
   }
 
   if (snapshotMatches && snapshotMatches.length) {
```

Comment:
Instead of using `!` can we just explicitly check `snapshotMatches[0]` instead of `.length` ?

Reply:
Yep, that seems reasonable for these.

Reply:
I accidentally applied this change to the last branch in the stack, so that's where it'll show up.